src/app/navbar/navbar.component.html

```diff
@@ -11,19 +11,46 @@
     <a class="navbar-item"></a>
   </div>
   <div class="navbar-end">
-    <a class="navbar-item">
-      <span [ngClass]="ping_info.is_alive?'tag is-success':'tag is-danger'">{{ping_info.service_status}}</span>
+    <a class="navbar-item is-hidden-mobile">
+      <span [ngClass]="ping_info.is_alive?'tag is-success':'tag is-danger'">
+        <i [ngClass]="ping_info.is_alive?'fa fa-check-circle-o':'fa fa-times-circle'" aria-hidden="true"></i>
+      </span>
+      <span class="tag" style="white-space: pre">{{ping_info.service_status}}</span>
     </a>
-    <a class="navbar-item">
-      <button *ngIf="ping_info.is_alive" class="button is-danger is-small" (click)="openModal()">
+    <div class="navbar-item has-dropdown is-hoverable is-hidden-mobile">
+      <a *ngIf='isUserLoggedIn' class="navbar-link">
+        <b class="has-text-grey"> &nbsp;{{userName}}&nbsp; </b>
+      </a>
+      <div class="navbar-dropdown">
+        <a class="navbar-item">
+          <span *ngIf='isUserLoggedIn' (click)="logout()">
+            <i class="fa fa-sign-out" aria-hidden="true"></i>&nbsp; Log Out
+          </span>
+        </a>
+        <a class="navbar-item">
+          <span *ngIf="ping_info.is_alive && isUserLoggedIn || isSkip" (click)="openModal()">
+            <i class="fa fa-power-off" aria-hidden="true"></i>&nbsp; Shut down
+          </span>
+        </a>
+      </div>
+    </div>
+    <div class="is-hidden-desktop navbar-dropdown">
+      <span [ngClass]="ping_info.is_alive?'tag is-success':'tag is-danger'">{{ping_info.service_status}}</span>
```

Comment:
in case of mobile view, just show icon tag for up and down status

Reply:
Yes already doing the same thing. This is the desktop section you are looking into. for mobile section check `line 37` n below